docs-src/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/cobaltspeech/sdk-diatheke/docs-src
+
+go 1.15
+
+require github.com/cobaltspeech/docdock v0.0.0-20201021030030-709475da43cc // indirect
```

Comment:
same thing as in sdk-cubic. If this is supposed to be here, good

Reply:
yes, this is how hugo modules work--it allows us to bring in the theme without copying it into the repo.